practical-07/src/components/HomeView.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+<div>
+  <div class="row">
+     <div>
+        <button class="add-btn" size="sm" pill variant="outline-info" @click="handleModel(true)">
+          <i class="fa-solid fa-file-circle-plus"></i>
+          <span class="separator"/>
+          <span class="text">Add car</span>
+        </button>
+     </div>
+    <div class="column">
+      <TransitionGroup name="list" tag="">
+      <div v-for="carItem in carData" :key="carItem.name" class="dataContainer">
+        <GalleryCard 
+          :carImage="carItem.image"
+          :carName="carItem.name"
+          :carDetails="carItem.details"
+          :carPrice="carItem.price"
```

Comment:
Codacy has a fix for the issue: [Insert `····`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
              :carPrice="carItem.price"
```